spec/support/features/document_capture_step_helper.rb

```diff
@@ -1,20 +1,31 @@
 module DocumentCaptureStepHelper
+  def wait_for_form_page_load
+    wait_for_content_to_disappear do
+      expect(page).not_to have_content(t('doc_auth.headings.interstitial'), wait: 10)
+    end
+  end
+
   def submit_images
     click_on 'Submit'
 
     # Wait for the the loading interstitial to disappear before continuing
-    wait_for_content_to_disappear do
-      expect(page).not_to have_content(t('doc_auth.headings.interstitial'), wait: 10)
-    end
+    wait_for_form_page_load
   end
 
   def continue_doc_auth_form
     click_on 'Continue'
 
     # Wait for the the loading interstitial to disappear before continuing
-    wait_for_content_to_disappear do
-      expect(page).not_to have_content(t('doc_auth.headings.interstitial'), wait: 10)
+    wait_for_form_page_load
+  end
+
+  def attach_and_submit_images_for_split_doc_auth(with_selfie: false)
```

Comment:
this method does not appear to be necessary based on your usage in analytics.spec:1260 ... looks like you can get rid of this method and call _#attach_and_submit_images_ instead